src/main/scala/cromwell/binding/values/WdlFileValue.scala

```diff
@@ -0,0 +1,25 @@
+package cromwell.binding.values
+
+import java.io.File
+import cromwell.binding.types.WdlFileType
+
+case class WdlFileValue(value: File) extends WdlPrimitiveValue {
```

Comment:
This whole structure kind of screams typeclass to me however the real power of typeclasses is making it easy to add new types down the road which seems extremely unlikely for us. So throwing it out there but probably best to ignore